### PR TITLE
#45 Fix autemated test

### LINKED ISSUE #45: Fix autemated test
When I wrote the result of the automated test to a file, the same answer was written twice like below.
`minishell> a | b  
content: a attr: 0  
content: | attr: 2  
content: b attr: 0
content: a attr: 0
content: | attr: 2
content: b attr: 0`

### PR DESCRIPTION
**Pull request by Hiroaki-K4**
When I wrote the result of the automated test to a file, the same answer was written twice like below.
minishell> a | b content: a attr: 0 content: | attr: 2 content: b attr: 0 content: a attr: 0 content: | attr: 2 content: b attr: 0

Two processes were created by fork () in the run_command function. The run_command function is a function I made a long time ago, so I need to modify it. Fixed a bug by commenting it out and not using it.

### CODE REVIEW

**Comment by Hiroaki-K4** [comment]
I also  added sysntax error test.

**Review by cheedah — COMMENTED** [review]
test/result/下がいらないくらいで、他は大丈夫だと思います

**Comment by Hiroaki-K4** [comment]
テスト実行時にmkdirでresultフォルダを生成するように修正。

File: srcs/tokenize.c [hunk 1]
```diff
@@ -99,7 +99,7 @@ void	tokenize(char *line, t_list **token_list)
 	int		pos;
 
 	pos = 0;
-	while (line[pos])
+	while (line[pos] && pos < (int)ft_strlen(line))
```
Comment (cheedah) [inline srcs/tokenize.c]: intにキャストするよりはposをsize_tにする方が用途がはっきりしていいかも

File: test.sh [hunk 2]
```diff
@@ -23,11 +23,23 @@ make fclean > /dev/null
...
 # Test Tokenizer
-rm -f result.txt
+rm -f test/result/test_tokenizer.txt
+mkdir test/result
 make > /dev/null
 while read line
 do
-	eval ${line} >> result.txt
+	eval ${line}
+	eval ${line} >> test/result/test_tokenizer.txt
```
Comment (cheedah) [inline test.sh]: 同じ処理が2回走るのはイケテナイ

**Review by cheedah — APPROVED** [review]
あとはいいと思います

File: srcs/tokenize.c (hunk 1 above)
Comment (Hiroaki-K4) [reply srcs/tokenize.c]: 修正完了。

File: test.sh (hunk 2 above)
Comment (Hiroaki-K4) [reply test.sh]: teeコマンドを使って1行にまとめた。

**Pull request merged**